src/tests/p11test/p11test_case_common.c

```diff
@@ -1213,11 +1213,23 @@ const char *get_mechanism_flag_name(int mech_id)
 		case CKF_EC_ECPARAMETERS:
 			return "CKF_EC_ECPARAMETERS";
 		default:
-			sprintf(flag_buffer, "0x%.8X", mech_id);
-			return flag_buffer;
+			return "unknown";
```

Comment:
I think we have most of the flags already covered now, but if there is some new, this hugely simplified adding the missing flag, while now, the word "unknown" wont be much useful. I would propose to keep the old behavior (while it will need a new static buffer).